packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -52,6 +64,24 @@ private void loadUrl(MethodCall methodCall, Result result) {
     result.success(null);
   }
 
+  private void canGoBack(MethodCall methodCall, Result result) {
+    result.success(webView.canGoBack());
+  }
+
+  private void canGoForward(MethodCall methodCall, Result result) {
+    result.success(webView.canGoForward());
+  }
+
+  private void goBack(MethodCall methodCall, Result result) {
+    webView.goBack();
+    result.success(null);
+  }
+
+  private void goForward(MethodCall methodCall, Result result) {
+    webView.goForward();
```

Comment:
Same, lets only do this if the webview can go forward (and complete with success anyway).

Reply:
d0e68de